docs/get-started/configuration.md

```diff
@@ -557,6 +560,11 @@ of v0.3.0:
     "enabled": true,
     "target": "local",
     "otlpEndpoint": "http://localhost:4317",
+    "otlpProtocol": "http",
+    "otlpHeaders": {
+      "Authorization": "Bearer ${MY_TOKEN}",
```

Comment:
when using token authentication you need to consider token expiry. look, for example, at the GCP OLTP sample code:

https://docs.cloud.google.com/stackdriver/docs/instrumentation/migrate-to-otlp-endpoints#auth-headers